pulsar-websocket/src/main/java/org/apache/pulsar/websocket/admin/WebSocketWebResource.java

```diff
@@ -62,14 +64,15 @@ protected WebSocketService service() {
      * @return the web service caller identification
      */
     public String clientAppId() {
-        if (clientId != null && service().getConfig().isAuthenticationEnabled()) {
+        if (isBlank(clientId) && service().getConfig().isAuthenticationEnabled()) {
```

Comment:
if `authentication` is disabled and clientId is available in `httpRequest` then this will still return null `clientId` even it's present into `httpRequest`. Should we still return correct `clientId` in this case for logging?

Reply:
I have modified `WebSocketWebResource#clientAppId()` to try acquiring `clientId` even when authentication is disabled. Is my understanding correct?